src/overlays/actors/ovl_En_Stone_heishi/z_en_stone_heishi.h

```diff
@@ -2,16 +2,33 @@
 #define Z_EN_STONE_HEISHI_H
 
 #include "global.h"
+#include "assets/objects/object_sdn/object_sdn.h"
 
 struct EnStoneheishi;
 
 typedef void (*EnStoneheishiActionFunc)(struct EnStoneheishi*, PlayState*);
 
 typedef struct EnStoneheishi {
     /* 0x000 */ Actor actor;
-    /* 0x144 */ char unk_144[0x110];
+    /* 0x144 */ SkelAnime skelAnime;
+    /* 0x188 */ Vec3s jointTable[SOLDIER_LIMB_MAX];
+    /* 0x1EE */ Vec3s morphTable[SOLDIER_LIMB_MAX];
     /* 0x254 */ EnStoneheishiActionFunc actionFunc;
-    /* 0x258 */ char unk_258[0x6C];
+    /* 0x258 */ s16 headRotY;
+    /* 0x25A */ s16 headRotX;
+    /* 0x25C */ s16 headRotZ;
+    /* 0x25E */ s16 targetHeadPosY;
+    /* 0x260 */ s16 targetHeadPosX;
```

Comment:
These are also rotations too, not positions, and there's room to make it another Vec3s.